src/gsi/cplr_read_wrf_mass_guess.f90

```diff
@@ -2240,6 +2297,74 @@ subroutine read_wrf_mass_netcdf_guess_wrf(this,mype)
                      tsn=ges_tv_it(j,i,k)/(one+fv*max(zero,ges_q_it(j,i,k)))
                      call hx_dart(ges_qr(j,i,k),ges_qg(j,i,k),ges_qs(j,i,k),ges_rho,tsn,ges_dbz(j,i,k),.false.)
                    end if
+
+! --- direct reflectivity DA --- CV transform on hydrometeors / CVq, CVlogq, and CVpq
+! Power transform for related q variables begins here==============================================! g.zhao
```

Comment:
delete "========="

Reply:
I deleted these unnecessary codes according to your comment